### PR TITLE
Issue #3759

### PR DESCRIPTION
**Pull request by desmay**
#### Pull Request (PR) description
Check to validate if Team has an owner

#### This Pull Request (PR) fixes the following issues
Issue #3759

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 * EXODistributionGroup
   * Fixes the export of group membership to use Identity.
+* TeamsTeam
+  * Check if Teams owners are null
```
Comment (andikrueger) [inline CHANGELOG.md:8]: Could you please add the issue number to the changelog?

File: Modules/Microsoft365DSC/DSCResources/MSFT_TeamsTeam/MSFT_TeamsTeam.psm1 [hunk 2]
```diff
@@ -203,6 +203,10 @@ function Get-TargetResource
             Ensure                            = 'Present'
         }
 
+        if($null -ne $Owners){
+            $result.Add('Owner',$Owners[0].User.ToString())
```
Comment (andikrueger) [inline Modules/Microsoft365DSC/DSCResources/MSFT_TeamsTeam/MSFT_TeamsTeam.psm1:207]: This aligns with the previous implementation. Why are we only grabbing the first user and not all owners of a team?

**Review by andikrueger — COMMENTED** [review]
Just two minor comments

**Comment by ykuijs** [comment]
This PR has been replaced by #3865. Closing this PR.

**Pull request closed without merging**